chart/templates/vmsingle/deployment.yaml

```diff
@@ -20,6 +20,10 @@ spec:
         job: vmsingle
         {{- include "prometheus-benchmark.labels" . | nindent 8 }}
     spec:
+      {{- with $.Values.nodeSelector }}
```

Comment:
Will it work correctly when nodeSelector isn't specified?

Reply:
It will, yes